.changeset/pretty-zebras-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+'@itwin/itwinui-react': patch
+---
+
+The different density settings for table will now also affect horizontal spacing.
```

Comment:
this changeset is wrong

Reply:
Oops. Removed and added the right changeset.